index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7507e0edea.js" crossorigin="anonymous"></script>
+        <title>Personal portfolio</title>
+</head>
+<body>
+    <div>
+    <nav class="bar">
+       <a href="#"><h2 class="logo">My Logo</h2></a>
+        <div class="icon">
+            <a href="#" alt="bars"><i class="fa-solid fa-bars"></i></a>
+        </div>
+    </nav>
+</div>
+    <div>
+<header>
+    <h1 class="name">I’m JIPSON <br>
+        Glad to see you!</h1>
+  </header>
+        <p class="ABout"><b>I’m a software developer! I can help you build a product ,
+             feature or website Look through some of my work and experience!
+              If you like what you see and have a project you need coded, don’t hestiate to contact me.</b></p>
+        </div>
+            <br>
+                <h3>LET'S CONNECT</h3>
```

Comment:
- [ ] Kindly try to follow the design provided in Figma as much as possible, this includes font-family, font-size, color, margins and padding...

## Your design:
![image](https://user-images.githubusercontent.com/50721479/192095512-1eceb42e-ffcf-45c0-87cf-6c6ffe57e0be.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/192095519-056d9012-ec9f-4862-a66a-3a63e1bb5b12.png)